themes/default/components/src/components/date-countdown/date-countdown.tsx

```diff
@@ -47,12 +52,18 @@ export class DateCountdown {
     }
 
     private renderCountdown() {
-        const { days, hours, minutes, seconds } = this.countdownData;
+        const { days, hours, minutes, seconds, distance } = this.countdownData;
         return <p class={this.textClass}>
-            <span>{ days }{this.renderValueLabel("days")} </span>
-            <span>{ hours }{this.renderValueLabel("hours")} </span>
-            <span>{ minutes }{this.renderValueLabel("minutes")} </span>
-            <span>{ seconds }{this.renderValueLabel("seconds")}</span>
+            { distance > 0 ?
```

Comment:
YES. 😂 Completely forgot to come back to this one. 

As a pure nit, I might suggest changing the name to like `remainingTimeInMS` or something more representative of what the value is, as `distance` sounds less like time and more like space. 😄

Reply:
😄